assets/app/scripts/controllers/settings.js

```diff
@@ -9,61 +9,121 @@
  * Controller of the openshiftConsole
  */
 angular.module('openshiftConsole')
-  .controller('SettingsController', function ($scope, DataService, AlertMessageService, $filter, $modal, $location, LabelFilter, $timeout, Logger) {
+  .controller('SettingsController', function ($routeParams, $scope, DataService, AlertMessageService, $filter, $modal, $location, LabelFilter, $timeout, Logger, ProjectsService) {
+    $scope.projectName = $routeParams.project;
     $scope.quotas = {};
     $scope.limitRanges = {};
     $scope.limitsByType = {};
     $scope.labelSuggestions = {};
-    $scope.alerts = $scope.alerts || {};
+    $scope.alerts = {};
+    $scope.renderOptions = {
+      hideFilterWidget: true
+    };
     $scope.emptyMessageQuotas = "Loading...";
     $scope.emptyMessageLimitRanges = "Loading...";
-    $scope.renderOptions = $scope.renderOptions || {};
-    $scope.renderOptions.hideFilterWidget = true;
 
     var watches = [];
 
-    DataService.list("resourcequotas", $scope, function(quotas) {
-      $scope.quotas = quotas.by("metadata.name");
-      $scope.emptyMessageQuotas = "There are no resource quotas set on this project.";
-      Logger.log("quotas", $scope.quotas);
-    });
-
-    DataService.list("limitranges", $scope, function(limitRanges) {
-      $scope.limitRanges = limitRanges.by("metadata.name");
-      $scope.emptyMessageLimitRanges = "There are no resource limits set on this project.";
-      // Convert to a sane format for a view to a build a table with rows per resource type
-      angular.forEach($scope.limitRanges, function(limitRange, name){
-        $scope.limitsByType[name] = {};
+    ProjectsService
+      .get($routeParams.project)
+      .then(_.spread(function(project, context) {
+        $scope.project = project;
+        $scope.openDeleteModal = function() {
+        // opening the modal with settings scope as parent
+        var modalInstance = $modal.open({
```

Comment:
none of the delete project modal stuff should be there anymore, this was all changed to use the common Delete button